websites/D/DashNet/presence.ts

```diff
@@ -14,6 +14,20 @@ function presenceSet(): void {
 	}
 }
 
+function spaceAfterNumbers(str: string) {
+	let newStr = String(),
+		found = false;
+	for (const c of str.split("")) {
+		newStr +=
+			isNaN(Number(c)) && found == false && c !== "." && c !== ","
+				? ` ${c}`
+				: c;
+		if (newStr.includes(" ")) found = true;
+	}
+
+	return newStr;
+}
```

Comment:
this whole function is unnecessarily complex and is fixing an issue that shouldn't exist to begin with. In order to fix this you need to use the proper query selector when getting the text

Reply:
Fixed this with a new commit, replaced the old function with a more efficient query selector.